app/uk/gov/hmrc/eusubsidycompliancestub/models/NonHmrcSubsidy.scala

```diff
@@ -22,12 +22,10 @@ import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.eusubsidycompliancestub.models.types.{EORI, EisSubsidyAmendmentType, SubsidyAmount, SubsidyRef, TraderRef}
 
 case class NonHmrcSubsidy(
-  subsidyUsageTransactionId: Option[
-    SubsidyRef
-  ], // TODO if we can't get SCP09 sorted we'll need to uppercase ID in the format
```

Comment:
No action is needed. This TODO is from 2 years ago and doesn't reference an existing issue